<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>421</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tab_deduction">
     <attribute name="title">
      <string>规则</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plainTextEdit_deduction">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>411</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_first">
     <attribute name="title">
      <string>first集合</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plainTextEdit_first">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>411</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_follow">
     <attribute name="title">
      <string>follow集合</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plainTextEdit_follow">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>411</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_deducted_output">
     <attribute name="title">
      <string>推导结果</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plainTextEdit_deducted_output">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>411</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_eliminate">
     <attribute name="title">
      <string>简化规则</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plainTextEdit_deducted_output_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>411</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_input">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>321</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_input">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>输入 请将文法开始字符放在第一行</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tip_input">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>格式为 S -&gt; Ab</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_wait_deduction">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>310</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入待推导语句</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_wait_deduction">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>330</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_confirm">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>230</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_deducted">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>350</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>推导</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>240</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>epsilon  @</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
